### PR TITLE
Added new schema for form 21P-0969 Update Income and Assets

### PR DESCRIPTION
**Pull request by danlim715**
# New schema
Added a new schema for form 21P-0969 Update Income and Assets Evidence.

Incremented gem version to 22.3.0

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/82561

## Pull Requests to update the schema in related repositories

- https://github.com/department-of-veterans-affairs/vets-api/pull/
- https://github.com/department-of-veterans-affairs/vets-website/pull/

### CODE REVIEW

File: test/schemas/21P-0969/schema.spec.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-undef */
+import _ from 'lodash';
```
Comment (wayne-weibel) [inline test/schemas/21P-0969/schema.spec.js:2]: possible to import only used functions?

**Review by wayne-weibel — CHANGES_REQUESTED** [review]
One minor change, and some questions
Would like @scottsdevelopment to look this over also, since he is more familiar with json-schema

File: src/schemas/21P-0969/schema.js [hunk 2]
```diff
@@ -0,0 +1,326 @@
...
+    trusts: schemaHelpers.getDefinition('trusts'),
+    hasAnnuities: definitions.yesNoSchema,
+    annuities: schemaHelpers.getDefinition('annuities'),
+    hasUnreportedAssets: definitions.yesNoSchema,
+    unreportedAssets: schemaHelpers.getDefinition('unreportedAssets'),
+    hasDiscontinuedIncomes: definitions.yesNoSchema,
+    discontinuedIncomes: schemaHelpers.getDefinition('discontinuedIncomes'),
+    hasIncomeReceiptWaivers: definitions.yesNoSchema,
+    incomeReceiptWaivers: schemaHelpers.getDefinition('incomeReceiptWaivers'),
+    statementOfTruthCertified: {
```
Comment (TaiWilkin) [inline src/schemas/21P-0969/schema.js:303]: Should this be a boolean?

Comment (danlim715) [reply src/schemas/21P-0969/schema.js:303]: ah, yes. nice catch!

**Review by TaiWilkin — APPROVED** [review]
Thanks for updating that!

**Review by ataker — APPROVED** [review]

**Review by wayne-weibel — APPROVED** [review]

**Pull request merged**